grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -626,27 +594,25 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
     );
   }
 
-  getApplyLabelFilterClickHandler = (label: LabelKeyValue) => {
+  applyLabelFilterClickHandler = (label: LabelKeyValue) => {
     const {
       store: { filtersStore },
     } = this.props;
 
-    return () => {
-      const {
-        filters: { label: oldLabelFilter = [] },
-      } = this.state;
+    const {
+      filters: { label: oldLabelFilter = [] },
+    } = this.state;
 
-      const labelToAddString = `${label.key.id}:${label.value.id}`;
-      if (oldLabelFilter.some((label) => label === labelToAddString)) {
-        return;
-      }
+    const labelToAddString = `${label.key.id}:${label.value.id}`;
```

Comment:
This part looks like a duplicate of what's below in `Integrations.tsx` - can it be subject to deduplication maybe? like a common helper for filter labels or something simillar?

Reply:
@teodosii I moved duplicated behaviour to common filters store: https://github.com/grafana/oncall/pull/3383/commits/e07fb1c05523becdac25a7b3da88a386de62a8b6